.dockerignore

```diff
@@ -0,0 +1,3 @@
+*
+!docker/
+!pkg/docker
```

Comment:
I'm not sure I follow what you want here.  There is an upstream `docker` directory which I preferred to leave alone, so I put the docker `build` file in `pkg/docker/build` instead of `docker/build`.

That means that I needed to ignore both the upstream `docker/` dir and the added `pkg/docker` dir.

Is that correct, or would you prefer me to do something else?